frontend/packages/client/src/components/Community/ProposalThresholdEditor/FormConfig.js

```diff
@@ -30,7 +30,11 @@ const Schema = (isValidFlowAddress) =>
       .when('onlyAuthorsToSubmitProposals', {
         is: true,
         then: addresValidation(yup.string(), isValidFlowAddress, true),
-        otherwise: addresValidation(yup.string(), isValidFlowAddress, false),
+        otherwise: addresValidation(
```

Comment:
it looks like this is missing an s - addres**s**Validation

Reply:
thanks Jonathan! getting it fixed!